tests/dummy/app/routes/application.js

```diff
@@ -3,6 +3,10 @@ import Ember from 'ember';
 export default Ember.Route.extend({
     store: Ember.inject.service(),
     session: Ember.inject.service(),
+    moment: Ember.inject.service(),
+    beforeModal() {
```

Comment:
should be `beforeModel()` (typo)

I wonder if this was the cause of the earlier test errors!